res/lang/stats/stats_en.txt

```diff
@@ -56,19 +56,19 @@ FINANCES_SPONSORSHIP_INCOME__PENALTY = Sponsoring: Income / Penalty
 FINANCES_STATIONS                    = Stations
 FINANCES_STATIONS_FEES               = Stations: Maintenance
 FINANCES_STATIONS_BUY_SELL           = Stations: Sold / Bought
-FINANCES_CREDIT_TAKEN__REPAYED       = Credit: New / Repayed
+FINANCES_CREDIT_TAKEN__REPAYED       = Loans: New / Repayment
```

Comment:
"New Loans / Repayment" or "New / Payed back"

Reply:
going with first option for now